users/tests.py

```diff
@@ -1,3 +1,171 @@
 # from django.test import TestCase
+import pytest
+from users.models import Profile
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.utils import timezone
 
-# Create your tests here.
+
+CREATION_DATE = timezone.now()
+
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+PHONE_NUMBER = "0541234567"
+LONG_PROFESSION = " test test test test test test test test test test test test test test test test" \
+                 " test test test test test test test test test" \
+                 " test test test test test test test test test" \
+                 " test test test test "
+LONG_BIO = """ test test test test test test test test test test test test test test test test
```

Comment:
what are we trying to achieve using this long text?

Reply:
bio max length is 500 and max profession length is 100 so this long text is to exceed the bio max length

Reply:
I would rather have here something like `"test_exceeding_max_length"*100` or something like this